packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -213,10 +213,20 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
     actions.workspace.openDefaultItem();
   } catch (error) {
     state.editor.notFound = true;
-    let detail = error.response?.data?.errors?.detail;
+    const errors = error.response?.data?.errors;
+    let detail = errors?.detail;
+    if (error.response.status === 422) {
+      state.editor.unprocessableEntityError = true;
```

Comment:
Will we put this one to false at any point later? Maybe @christianalfoni has a good suggestion for this.

Reply:
Oh true, should set it to false on enter that function